pkg/arvo/sys/zuse.hoon

```diff
@@ -2373,6 +2376,19 @@
             [%disavow ~]
         ==  ==
       ::
+      ++  udiff-to-diff
+        |=  [=a=udiff =a=point]
+        ^-  diff
+        ?-    +<.a-udiff
+            %disavow  ~|(%udiff-to-diff-disavow !!)
+            %rift     [%rift rift.a-point rift.a-udiff]
+            %spon     [%spon sponsor.a-point sponsor.a-udiff]
+            %keys
+          :+  %keys
+            [life.a-point (~(gut by keys.a-point) life.a-point *[@ud pass])]
+          [life crypto-suite pass]:a-udiff
+        ==
+      ::
       ++  inverse
```

Comment:
Do we even need this implementation, or could it just be +flop? Seems like there aren't any cases where it modifies its input.

Reply:
ah, whoops, it's supposed to flip the `to` and `from` for each diff individually as well.  I had that initially, but I used the `[|2 &2]` syntax which @Fang- complained about as unusual.  He was justified by me totally misreading it as *not* flipping them in the "fix".  I'll switch it to use the actual faces, eg `[%rift to from]:diff`.